instance/tests/integration/factories/instance.py

```diff
@@ -79,8 +79,8 @@ def create(cls, *args, **kwargs):  # pylint: disable=arguments-differ
     # or a release candidate tag will work.  We point both the edx-platform and the configuration
     # versions to the branch "integration" in our own forks.  These branches are based on the
     # corresponding openedx_release versions from upstream, but can contain custom modifications.
-    openedx_release = 'open-release/koa.2a'
+    openedx_release = 'open-release/koa.3'
     configuration_source_repo_url = 'https://github.com/open-craft/configuration.git'
-    configuration_version = 'integration-koa'
+    configuration_version = 'integration-koa.3'
```

Comment:
Is this needed if/when the integration-koa branch is rebased on koa.3?

Reply:
@janimo, `integration-koa` branch is used for integration tests right now. I realized that there is no sense to rebase it.